eg01-eg23/eg01-eg23-cycamore.xml

```diff
@@ -1,279 +1,303 @@
-            <commod>FR Reprocessed</commod>
+            <commod>mox_U</commod>
             <info>
-              <buf_size>1e+100</buf_size>
+              <buf_size>1E100</buf_size>
               <efficiencies>
                 <item>
-                  <comp>Pu</comp>
-                  <eff>0.99</eff>
+                  <comp>U</comp> <eff>.998</eff>
                 </item>
               </efficiencies>
             </info>
           </item>
         </streams>
-        <throughput>1e+100</throughput>
       </Separations>
     </config>
-    <name>FR Reprocess</name>
   </facility>
 
-
   <facility>
     <config>
-      <Reactor>
-        <assem_size>11666.7</assem_size>
-        <cycle_time>18</cycle_time>
-        <fuel_incommods>
-          <val>LWR Fuel</val>
-        </fuel_incommods>
-        <fuel_inrecipes>
-          <val>fresh_uox</val>
-        </fuel_inrecipes>
-        <fuel_outcommods>
-          <val>LWR Spent Fuel</val>
-        </fuel_outcommods>
-        <fuel_outrecipes>
-          <val>spent_uox</val>
-        </fuel_outrecipes>
-        <fuel_prefs>
+      <FuelFab>
+        <fill_commods>
+          <val>tailings</val>
+        </fill_commods>
+        <fill_recipe>depleted_u</fill_recipe>
+        <fill_size>1e+60</fill_size>
+        <fiss_commod_prefs>
           <val>1</val>
-        </fuel_prefs>
-        <n_assem_batch>1</n_assem_batch>
-        <n_assem_core>3</n_assem_core>
-        <power_cap>1000</power_cap>
-        <refuel_time>1</refuel_time>
-      </Reactor>
+          <val>2</val>
+        </fiss_commod_prefs>
+        <fiss_commods>
+          <val>uox_Pu</val>
+          <val>mox_Pu</val>
+        </fiss_commods>
+        <fiss_size>1e+60</fiss_size>
+        <outcommod>mox</outcommod>
+        <spectrum>fission_spectrum_ave</spectrum>
+        <throughput>1e+60</throughput>
+      </FuelFab>
     </config>
-    <lifetime>960</lifetime>
-    <name>LWR</name>
+    <name>mox_fuel_fab</name>
   </facility>
 
 
   <facility>
+      <name>uox_pool</name>
+      <!-- Spent Fuel Spends ~3 years (36 timesteps) -->
+      <!-- in NPP pools prior to process (carre_overview_2009) -->
+      <config>
+          <Storage>
+              <in_commods>
+                  <val>uox_waste</val>
+              </in_commods>
+              <out_commods>
+                  <val>cool_uox_waste</val>
+              </out_commods>
+              <residence_time>36</residence_time>
+              <throughput>1e299</throughput>
+              <max_inv_size>1e299</max_inv_size>
+          </Storage>
+      </config>
+  </facility>
+
+  <facility>
+      <name>sfr_mox_pool</name>
+      <!-- Spent Fuel Spends ~3 years (36 timesteps) -->
+      <!-- in NPP pools prior to process (carre_overview_2009) -->
+      <config>
+          <Storage>
+              <in_commods>
+                  <val>sfr_spent_mox</val>
+              </in_commods>
+              <out_commods>
+                  <val>cool_sfr_spent</val>
```

Comment:
This might be a poor answer, but the upcoming PR has all the scenarios standardized (all the commodity & recipe names are the same for all scenarios) and well-organized (hopefully!). Can we maybe revisit this then?